### ISSUE TITLE
Email delay problem with Postgrey

### ISSUE THREAD

**Issue by siorac** [schweikert/postgrey#50]
I have been using Postgrey on my server for several years without any problem.
But for 15 days, all the mails that I receive from @hotmail or @live senders, arrived with a delay of several hours.
I can not find a solution. I don't know where to change something on the configuration,
and what to change ?
Can anyone help me ?
Below, an example of the header of a message :

Return-Path: <test@hotmail.fr>
X-Original-To: toto@abc.fr
Delivered-To: toto@abc.fr
Received: from localhost (localhost [127.0.0.1])
    by monserveur.org (Postfix) with ESMTP id A68053035E5
    for <toto@abc.fr>; Sat, 8 Apr 2017 01:07:59 +0200 (CEST)
X-Virus-Scanned: Debian amavisd-new at monserveur.org
X-Spam-Flag: NO
X-Spam-Score: -99.95
X-Spam-Level:
X-Spam-Status: No, score=-99.95 tagged_above=-9999 required=2.8
    tests=[FREEMAIL_FROM=0.001, HTML_MESSAGE=0.07,
    RCVD_IN_MSPIKE_H3=-0.01, RCVD_IN_MSPIKE_WL=-0.01,
    SPF_HELO_PASS=-0.001, SPF_PASS=-0.001, TVD_SPACE_RATIO=0.001,
    USER_IN_WHITELIST=-100] autolearn=disabled
Received: from monserveur.org ([127.0.0.1])
    by localhost (monserveur.org [127.0.0.1]) (amavisd-new, port 10024)
    with ESMTP id W2PetLo_jCH2 for <toto@abc.fr>;
    Sat, 8 Apr 2017 01:07:59 +0200 (CEST)
X-Greylist: delayed 90370 seconds by postgrey-1.34 at monserveur.org; Sat, 08 Apr 2017 01:07:58 CEST
Received: from EUR02-HE1-obe.outbound.protection.outlook.com (mail-oln040092068051.outbound.protection.outlook.com [40.92.68.51])
    by monserveur.org (Postfix) with ESMTPS id EB7AC3035E4
    for <toto@abc.fr>; Sat, 8 Apr 2017 01:07:58 +0200 (CEST)
Received: from VE1EUR02FT055.eop-EUR02.prod.protection.outlook.com
    (10.152.12.51) by VE1EUR02HT189.eop-EUR02.prod.protection.outlook.com
    (10.152.13.113) with Microsoft SMTP Server (version=TLS1_2,
    cipher=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384_P384) id 15.1.1005.5; Thu, 6 Apr
    2017 12:28:52 +0000
Received: from DB6PR1001MB1223.EURPRD10.PROD.OUTLOOK.COM (10.152.12.52) by
    VE1EUR02FT055.mail.protection.outlook.com (10.152.13.34) with Microsoft SMTP
    Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256_P256) id
    15.1.1005.5 via Frontend Transport; Thu, 6 Apr 2017 12:28:52 +0000
Received: from DB6PR1001MB1223.EURPRD10.PROD.OUTLOOK.COM ([10.171.77.11]) by
    DB6PR1001MB1223.EURPRD10.PROD.OUTLOOK.COM ([10.171.77.11]) with mapi id
    15.01.1019.021; Thu, 6 Apr 2017 12:28:52 +0000

**Comment by deoren** [comment]
Sorry to ask the obvious question, but have you looked through your mail log file(s) to see what reason was given for greylisting the remote server(s)?

**Comment by schweikert** [comment]
Very likely caused by #32, fixed in Postgrey 1.37. Workaround: whitelist in postfix using check_client_access before postgrey.

**Comment by siorac** [comment]
Hello,
I still have not managed to find a solution with the delay problem.
Where can I send a message to the mailing-list ? (I didn't find it)

And I tried to stop postgrey,  I wrote the following commande :
service postgrey stop
the answer is that postgrey is stopped,   but in reality, postgrey continue to work !
And I don't know how to stop it.
Because, if I don't find a solution with the delay problem with all the hotmail, live, and office 365 addresses,   it will be necessary that I stop postgrey.

I thank you for your help.
